diff --git a/node_modules/vanta/src/vanta.rings.js b/node_modules/vanta/src/vanta.rings.js
index 8b92b04..22970bc 100644
--- a/node_modules/vanta/src/vanta.rings.js
+++ b/node_modules/vanta/src/vanta.rings.js
@@ -50,7 +50,7 @@ class Rings extends VantaBase {
 
     if (radius < 1) { radius = 1 }
     const extrudeSettings = {
-      amount: 0.4,
+      depth: 0.4,
       bevelEnabled: false,
       steps: 1,
       curveSegments: ~~((64 * ang) / 6.14)
